docs/Contributing.italian.md

```diff
@@ -0,0 +1,103 @@
+# Contribuire a React Native WebView
+
+Innanzitutto, _grazie_ per aver considerato di contribuire alla comunità di React Native. I pacchetti supportati dalla comunità sono possibili solo grazie a persone straordinarie come te.
+
+In secondo luogo, desideriamo che l'esperienza di contribuzione sia il migliore possibile. Nonostante siamo un piccolo team composto interamente da volontari, siamo felici di ricevere feedback sulla tua esperienza e se possiamo migliorare la documentazione o l'esperienza, ti preghiamo di farcelo sapere.
+
+## Come testare le modifiche
+
+Dopo aver forkato il reposito, clonalo sulla tuo computer e apporta le modifiche. Successivamente, potrai testarle in un'applicazione.
+
+Ci sono due metodi per testare:
+1) testare all'interno di una copia di react-native-webview;
+2) testare in un nuovo progetto creato con `react-native init`.
+
+### Testare in react-native-webview
+
+#### Per tutte le piattaforme:
+
+```sh
+yarn install
+```
+
+#### Per Android:
+
+```sh
+yarn android
+```
+
+L'applicazione di esempio per Android verrà compilata, Metro bundler si avvierà e l'applicazione verrà installata e avviata nell'emulatore Android.
+
+#### Per iOS:
+
+```sh
+pod install --project-directory=ios
```

Comment:
https://github.com/react-native-webview/react-native-webview/pull/2640 hasn't been merged yet, but it included pointing out that this should probably be `--project-directory=example/ios` (same presumably applies for macos)

Reply:
No worries, @TheAlmightyBob. I updated it as well as the Portuguese version. I left the original untouched so to avoid potential conflicts.